kura/org.eclipse.kura.ai.triton.server/src/main/java/org/eclipse/kura/ai/triton/server/MetricsParser.java

```diff
@@ -0,0 +1,91 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Eurotech and/or its affiliates and others
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *  Eurotech
+ ******************************************************************************/
+package org.eclipse.kura.ai.triton.server;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+public class MetricsParser {
+
+    private final List<String> rawMetrics;
+
+    public MetricsParser(List<String> metrics) {
+        this.rawMetrics = metrics;
+    }
+
+    /*
+     * Parse the metrics provided by a Triton Server. An example of metrics are the following:
+     * 
+     * # HELP nv_inference_compute_output_duration_us Cumulative inference compute output duration in microseconds (does
+     * not include cached requests)
+     * # TYPE nv_inference_compute_output_duration_us counter
+     * nv_inference_compute_output_duration_us{model="preprocessor",version="1"} 851348
+     * nv_inference_compute_output_duration_us{model="identity_long",version="1"} 0
+     * nv_inference_request_failure{model="preprocessor",reason="OTHER",version="1"} 0
+     * nv_pinned_memory_pool_used_bytes 0
+     * 
+     * The lines beginning with a # are filtered. The metric names have the following format:
+     * 
+     * <metric_name>-<model-name>-<parameter-1>-<parameter-2>...
+     * 
+     * For example the metric
+     * 
+     * nv_inference_request_failure{model="preprocessor",reason="OTHER",version="1"} 0
+     * 
+     * is converted to
+     * 
+     * nv_inference_request_failure-preprocessor-OTHER-1 0
```

Comment:
Is there a reason why we need to convert the metrics this way? (thinking about it it's probably the `{` right?)... as a user I would prefer to only learn one syntax. Having a middle man (i.e. Kura in this case) changing the metrics syntax arbitrarily is really annoying.